examples/app-playwright/test/basic.test.ts

```diff
@@ -1,14 +1,6 @@
-import { fileURLToPath } from 'node:url'
-import { test } from '@playwright/test'
-import { setup, url } from '@nuxt/test-utils/e2e'
-
-await setup({
-  rootDir: fileURLToPath(new URL('../', import.meta.url)),
-  // TODO: auto-detect playwright
-  runner: 'playwright'
-})
+import { expect, test } from '../customFixtures'
 
 test('test', async ({ page }) => {
-  await page.goto(url('/'))
-  test.expect(await page.getByRole('heading').innerText()).toBe('Welcome to Playwright!')
+  await page.goto('/')
+  await expect(page.getByRole('heading')).toHaveText('Welcome to Playwright!')
```

Comment:
These assertions will automatically retry over and over again until the condition is met or the timeout has been reached (they are non-flaky).